cfpb/cfgov-refresh
# Spanish updates

Makes spelling corrections on Spanish home page and changes instances of `Presentar una queja` to `Enviar una queja`

## Changes

- Changes `Presenter una queja` to `Enviar una queja` in mega menu, header, and Spanish home page section
- Fixes spelling of `millones` and `financiero` in cards on Spanish home page

## Testing

1. Check out branch and create Spanish home page
2. Verify spelling corrections & `Enviar una queja` usage

## Screenshots

![home-page-es-updates](https://user-images.githubusercontent.com/778171/57937478-e2ec7000-787a-11e9-8117-afd711e05e91.png)
